Interaction - Missions: Chains of Promathia Chapter 7

claywar opened the pull request:
<!-- remove space and place 'x' mark between square [] brackets or click the checkbox after saving to affirm: -->
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

claywar committed:
Interaction - Mission: Chains and Bonds

claywar committed:
Interaction - Mission: Flames in the Darkness

claywar committed:
Interaction - Mission: Fire in the Eyes of Men


Add missing titles

claywar committed:
Interaction - Mission: Calm Before the Storm

claywar committed:
[chore] Update enum name for CoP Mission: The Warrior's Path


Remove Sueleen Calm before the Storm events

claywar committed:
Interaction - Mission: The Warrior's Path


Formatting consistency


Additional cleanup, remove Iron Gate being a teleport source

The-Aerec commented:
@claywar - need help running through this and testing the interactions? I have some time to run through this commit

claywar commented:
It's been a busy week, but appreciate any feedback.  Did leave some notes in the scripts for things that stood out while converting.

The-Aerec commented:
Going to dump comments (They may or may not be relative):

Interacting with CoP6-4 to test the zone out:

Was given pretty much all available BCNMS

![image](https://user-images.githubusercontent.com/6257062/166594357-74b6b766-c92d-4bcb-a024-5c374e158ea5.png)

The-Aerec commented:
7-1 Interactions all checked: Working as intended.

The-Aerec commented:
7-2 Interactions all checked: Working as intended.

The-Aerec commented:
CoP 7-3: Post Mine Shaft event position Correct 

Timer other interactions work as intended.

The-Aerec commented:
7-4: Getting

![image](https://user-images.githubusercontent.com/6257062/166608244-1d06eb57-756a-4cde-92fa-9711bd91af11.png)

At ??? For Dulhan
At ??? For Cryptonberries
At Storage compartment

(Skipped the rest for now)

The-Aerec commented:
7-5: Interactions working as intended, same BCNM Entry issue as 6-4

Xaver-DaRed commented:
About the BCNM entrance.
That will happen if your character is a GM. Its one of those GM perks

The-Aerec commented:
Thats a nice upgrade~

claywar commented on scripts/missions/cop/7_5_The_Warriors_Path.lua:
```diff
@@ -0,0 +1,150 @@
+            onEventFinish =
+            {
+                [1] = function(player, csid, option, npc)
+                    -- Nag'molada steals one random light
```
Note to self: This isn't a random light, but aligns with player race.
![image](https://user-images.githubusercontent.com/81713309/167017989-e941a2b8-f005-4e62-af56-1a3903eb794d.png)

Galka:
```
[16:36:47] You obtain the light of Al'Taieu, only to have it stolen by Nag'molada...
```

WinterSolstice8 replied:
The Galka message is also a totally different one than the non-galka, because you never "obtain" any light. 

Galka:
Al'Taieu given & stolen, one line

Hume:
Vazhl taken (line 1)
Al'Taieu given (line 2)

claywar commented:
Tested completely, should be good to go

claywar committed:
QC: Promathia Chapter 7

Fix errant bitwise operations in PM 7-4

Fix issue through CoP 7-4


Fix remaining issues from testing for PM Chapter 7


Update BCNM 1 and 2 mob spawn locations
